src/human_player_module.c

```diff
@@ -158,43 +158,32 @@ ret_type_e record_human_input(void)
  * 
  * @param input Value entered by the human during his turn
  * 
- * @return Card_t* Contains the mapped card name and color values
+ * @return Card_t the mapped card name and color values
  *         NULL If the input is invalid
  */
-Card_t *map_user_input(char *input)
+Card_t map_user_input(char *input)
```

Comment:
Why do we have to change the return type of this function here?

Reply:
I think not necessary to use a pointer return here, the malloc memory in this function should be removed, which results in memory leakage.